data/mods/gen2/items.ts

```diff
@@ -87,10 +87,10 @@ export const BattleItems: {[k: string]: ModdedItemData} = {
 	},
 	stick: {
 		inherit: true,
-		desc: "If held by a Farfetch'd, its critical hit ratio is always at stage 2. (25% crit rate)",
+		desc: "If held by a Farfetch\u2019d, its critical hit ratio is always at stage 2. (25% crit rate)",
 		onModifyCritRatioPriority: -1,
 		onModifyCritRatio(critRatio, user) {
-			if (user.species.name === 'Farfetch\'d') {
+			if (toID(user.species.name) === 'farfetchd') {
```

Comment:
```suggestion
			if (user.species.id === 'farfetchd') {
```

I think?

Reply:
I think the `name` check is specifically for Transform purposes. It might be equivalent here but I'm not sure.

Reply:
`Species` objects are immutable, so name and id should not be changed. I check - Ditto's baseSpecies stays the same (Ditto) after it transforms, but the species changes, so species.id should work

Reply:
Done.